std/csv.d

```diff
@@ -1130,6 +1145,7 @@ public:
      *       closed before data was empty, a conversion failed, or when the
      *       row's length does not match the previous length.
      */
+    // In the CsvReader.popFront() method (around line 1365-1401)
```

Comment:
Doesn't add anything, and will get out of date. Nor does it contribute to the API docs since its only a ``//``.